admin/services/restart.service.js

```diff
@@ -100,7 +100,7 @@ restartService.canAllPupilsRestart = async (pupilsList) => {
 
 restartService.canRestart = async pupilId => {
   const checkCount = await checkDataService.sqlGetNumberOfChecksStartedByPupil(pupilId)
-  const pupilRestartsCount = await pupilRestartDataService.count({ pupilId: pupilId, isDeleted: false })
+  const pupilRestartsCount = await pupilRestartDataService.sqlGetNumberOfRestartsByPupil(pupilId)
```

Comment:
this method is huge.  
we should leave as-is for now (due to deadlines), but should be considered for tech debt after the trials.
i think Jon has modified this in his branch.  can you please discuss with him.